File: pdpapp/PDP/Models/CuentaBancaria.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PDP.Models
{
    internal class CuentaBancaria
    {
        String Titular;
        private double Saldo;

        public CuentaBancaria(String Titular, double Saldo)
        {
            this.Titular = Titular;
            _ = Saldo > 0 ? this.Saldo = Saldo : this.Saldo = 0;
        }

        public void Depositar(double dinero)
        {
            if(dinero > 0) this.Saldo += dinero;
        }

        public void Retirar(double dinero)
        {
            if(dinero <= Saldo)this.Saldo -= dinero;
        }
    }
}

File: app/src/main/java/upb/pdp/models/CuentaBancaria.java
package upb.pdp.models;

public class CuentaBancaria {

    private String titular;
    private double saldo;

    public CuentaBancaria(String titular){
        this.titular = titular;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setSaldo(double saldo) {
        if (saldo >= 0) {
            this.saldo = saldo;
        }
    }
    public String getTitular() {
        return titular;
    }

    public void setTitular(String titular) {
        this.titular = titular;
    }

    public void depositar(double cantidad) {
        saldo += cantidad;
    }

    public void retirar(double cantidad) {
        if (cantidad > 0 && cantidad <= saldo) {
            saldo -= cantidad;
        } else {
            System.out.println("Fondos insuficientes o cantidad inválida");
        }
    }
}
